gulpfile.js

```diff
@@ -232,78 +236,78 @@ function jsdoc(options, done) {
   });
 }
 
-gulp.task('build:gh-pages', ['build:demoapp'], function (done) {
-  runSequence(
-    'jsdoc:generate',
-    [
-      'jsdoc:statics',
-      'jsdoc:link-current',
-    ],
-    done);
-});
-
-gulp.task('jsdoc:generate', function (done) {
+function ghPagesBuild() {
+  return series(demoAppApple, demoApp, generateJsdoc, parallel(jsdocStatistics, linkJsdoc));
+}
+
+function generateJsdoc(done) {
   jsdoc({
     configure: 'jsdoc/conf.json',
     destination: config.dist.jsdoc + VERSION,
     recurse: true,
     readme: config.jsdoc.readme,
     template: 'node_modules/jsdoc-template'
   }, done);
-});
+}
 
-gulp.task('jsdoc:statics', function () {
-  return gulp.src(['jsdoc/index.html']).pipe(gulp.dest(config.dist.jsdoc));
-});
+function jsdocStatistics() {
+  return src('jsdoc/index.html').pipe(dest(config.dist.jsdoc));
+}
 
-gulp.task('jsdoc:link-current', function (done) {
+function linkJsdoc(done) {
   var link = config.dist.jsdoc + 'current';
 
   if (fs.existsSync(link)) {
     del.sync(link);
   }
 
   fs.symlink(VERSION, config.dist.jsdoc + 'current', done);
```

Comment:
mental note to replace this with the `link` from L258

Reply:
Does this need to be worked on?

Reply:
https://github.com/braintree/braintree-web-drop-in/pull/505/files#diff-b9e12334e9eafd8341a6107dd98510c9R268